app/scripts/views/settings.js

```diff
@@ -156,6 +159,25 @@ function ($, modal, Cocktail, Session, BaseView, AvatarMixin,
       });
     },
 
+    _swapDisplayName: function () {
+      var account = this.getSignedInAccount();
+      var displayName = account.get('displayName');
+      var email = account.get('email');
+
+      var $cardHeader = $('.card-header');
```

Comment:
could we use ` this.$...` for all these selectors here? see other usage in this same file